src/components/card/README.md

```diff
@@ -0,0 +1,28 @@
+# Card
+
+## Usage
+
+```js
+import { Card } from '@commercetools-frontend/ui-kit';
+```
+
+#### Description
+
+Cards are used to display content and actions on a single topic.
+
+#### Usage
+
+```js
+<Card theme="light" type="raised">
+  <p>You'll never guess where I've been!</p>
+</Card>
+```
+
+#### Properties
+
+| Props       | Type     | Required | Values           | Default  | Description                       |
+| ----------- | -------- | :------: | ---------------- | -------- | --------------------------------- |
+| `theme`     | `string` |    -     | `light`, `dark`  | `light`  | background color of the card      |
+| `type`      | `string` |    -     | `raised`, `flat` | `raised` | The type of card                  |
+| `classname` | `string` |    -     | -                | -        | className passed to the container |
```

Comment:
Are we sure that we want that custom styles can be passed to this? Last time I spoke with Dom regarding this (classname in Collapsible Panel) we agreed that we should aim at some point to not have this in any UiKit component. Is really needed? 🤔 or what do you think?

Reply:
Two points

1) I don't think it's feasible for the `ui-kit` team to refactor every single usage of the Card component. 
2) Accepting a classname is actually really useful when you are using CSS in JS because otherwise it can be impossible to override styles without !important.

Reply:
> I don't think it's feasible for the ui-kit team to refactor every single usage of the Card component.

Fair enough 👌 

> Accepting a classname is actually really useful when you are using CSS in JS because otherwise it can be impossible to override styles without !important.

Yeah, I didn't say that it is not useful, I was just asking because of the last information about this I had 😅

Reply:
Yeah, in a perfect world we would have props that would deal with every scenario and we wouldn't need to accept a classname. But this requires doing some research into how `Card` is being used.

Maybe it's possible in the next iteration to drop support for `classname` in Card. 🤷‍♂️